packages/devextreme/testing/testcafe/tests/scheduler/viewOffset/markup/appointmentsOrdering.ts

```diff
@@ -0,0 +1,210 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { createWidget, disposeWidget } from '../../../../helpers/createWidget';
+import { insertStylesheetRulesToPage, removeStylesheetRulesFromPage } from '../../../../helpers/domUtils';
+import url from '../../../../helpers/getPageUrl';
+import Scheduler from '../../../../model/scheduler';
+import { generateAppointments } from '../../helpers/generateAppointments';
+
+fixture.disablePageReloads`Offset: Markup appointments ordering`
+  .page(url(__dirname, '../../../container.html'));
+
+const SCHEDULER_SELECTOR = '#container';
+const REDUCE_CELLS_CSS = `
+.dx-scheduler-cell-sizes-vertical {
+  height: 25px;
+}`;
+
+const APPOINTMENTS_TIME = [
+  { startTime: '04:00:00', endTime: '08:00:00' },
+  { startTime: '10:15:00', endTime: '16:15:00' },
+  { startTime: '17:05:00', endTime: '22:05:00' },
+];
+// NOTE: Appointment has increased length for testing purpose (it makes screenshots more obvious).
+const APPOINTMENTS_TIMELINE_TIME = [
+  { startTime: '04:00:00', endTime: '08:00:00', endDateShiftDays: 1 },
+  { startTime: '10:15:00', endTime: '16:15:00', endDateShiftDays: 1 },
+  { startTime: '17:05:00', endTime: '22:05:00', endDateShiftDays: 1 },
+];
+
+const RECURRENT_APPOINTMENTS = [
+  {
+    startDate: '2023-08-01T09:00:00',
+    endDate: '2023-08-01T13:00:00',
+    recurrenceRule: 'FREQ=HOURLY;INTERVAL=24',
+    text: 'Hourly 09-13',
+  },
+  {
+    startDate: '2023-08-01T15:00:00',
+    endDate: '2023-08-01T19:00:00',
+    recurrenceRule: 'FREQ=WEEKLY;BYDAY=MO,WE,TH,FR',
+    text: 'Daily 15-19',
+  },
+];
+
+const APPOINTMENTS = {
+  month: generateAppointments(
+    '2023-08-26',
+    '2023-10-08',
+    APPOINTMENTS_TIME,
+  ),
+  timelineMonth: generateAppointments(
+    '2023-08-31',
+    '2023-09-08',
+    APPOINTMENTS_TIMELINE_TIME,
+  ).concat(RECURRENT_APPOINTMENTS),
+};
+
+const getScreenshotName = (
+  viewType: string,
+  offset: number,
+  startDayHour: number,
+  endDayHour: number,
+  firstDay?: number,
+) => `view_markup_ordering-appts_${viewType}_offset-${offset}_start-${startDayHour}_end-${endDayHour}_first-day-${firstDay}.png`;
+
+[
+  { views: [{ type: 'month', firstDayOfWeek: 0 }], dataSource: APPOINTMENTS.month },
+  { views: [{ type: 'timelineMonth', firstDayOfWeek: 0 }], dataSource: APPOINTMENTS.timelineMonth },
```

Comment:
Minor comment.

I don't think that we need the "firstDayOfWeek" here.
Let's remove it :)